projects/nginx/fuzz/http_request_proto.proto

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+syntax = "proto2";
+// Proto description of the http request fuzzer input format
+
+message HttpProto{
+	required string request = 1;
+	required string reply = 2;
```

Comment:
what's the point of fuzzing the `reply`? Does it go through a complicated processing / parsing by nginx code?

If yes, would it be better to fuzz it in a separate fuzz target?

Reply:
Since Nginx works as proxy, we though it would be nice to fuzz both sides. Nginx does indeed do some parsing before sending it over. We thought it might be interesting to fuzz both sides together since how nginx deals with the reply depends on the request